community/kernel/src/main/java/org/neo4j/kernel/impl/index/labelscan/NativeLabelScanStore.java

```diff
@@ -389,7 +387,11 @@ private void dropStrict() throws IOException
             index.close();
             index = null;
         }
-        storeFileHandle().delete();
+        Optional<FileHandle> fileHandle = storeFileHandle();
+        if ( fileHandle.isPresent() )
```

Comment:
It will throw `NoSuchFileException` if file does not exist, this will be caught in `drop`.